docs/models.yml

```diff
@@ -1349,11 +1363,13 @@ group:
     to: meeting/default_group_id
     on_delete: PROTECT
     restriction_mode: A
+    required: false
   admin_group_for_meeting_id:
     type: relation
     to: meeting/admin_group_id
     on_delete: PROTECT
     restriction_mode: A
+    required: false
```

Comment:
Does this mean, a group can optionally be the default or admin group of one meeting or does this also mean, that a meeting can optionally have a default or an admin group?
I think, only the first one is right.